### PR TITLE
Update learing-to-learn. md

### PR DESCRIPTION
**Pull request by tyagipr**
Changes in lesson and supplement material sections. @manaswinidas

### CODE REVIEW

File: learning-to-learn/learning-to-learn.md [hunk 1]
```diff
@@ -36,17 +36,27 @@ https://www.youtube.com/watch?v=-71zdXCMU6A
...
-
-[Learning to Learn (slides)](https://docs.google.com/presentation/d/1pMkBP32lra-O_BS6kOqj81IzLspN1UtKT64jqOXS9Kc/edit?usp=sharing)
+- Learning to learn(The mind is a muscle)
+    - Learning Styles:
+      - Most people use combination of learning styles (e.g Seeing, Listening, Touching, Taking notes).
+    - How to know, what is our learning style?
+      - Fixing Information:
+         - Understanding that something is important, but it's not enough on its own.
+         - Hand-on practice truly helps you to grasp the concepts, it truly helps you automate the knowledge which you've acquired.
+         - Take breaks while learning or studying to alternate between focused and diffuse mode.
```
Comment (manaswinidas) [inline learning-to-learn/learning-to-learn.md]: Is it 'diffused'?

File: learning-to-learn/learning-to-learn.md [hunk 2]
```diff
@@ -36,17 +36,27 @@ https://www.youtube.com/watch?v=-71zdXCMU6A
...
+         - Do not procrastinate the things, use pomodoro method- work for just 20 minutes on something you don’t want to do, then take a break
+         - Test yourself during the studying process by look away and try to recall what you just studied.
```
Comment (manaswinidas) [inline learning-to-learn/learning-to-learn.md]: By 'looking' away and 'trying' to recall

Comment (tyagipr) [reply learning-to-learn/learning-to-learn.md]: PPT material has so many grammatical mistakes, if you take any line from there, you will usually get these errors. 
We need to correct those PPT material also, i guess.

**Comment by alodahl** [comment]
hi @tyagipr , since the issue I think you're working on was just about the materials & supplemental resources (#570 ? tell us next time), you can just focus on making your sections correct and skip everything else unless it is tiny.  Do both the outline and the slideshow both have a lot of errors?  If so, I can make that a separate issue.

File: learning-to-learn/learning-to-learn.md [hunk 3]
```diff
@@ -53,9 +53,9 @@ https://www.youtube.com/watch?v=-71zdXCMU6A
       - Fixing Information:
          - Understanding that something is important, but it's not enough on its own.
          - Hand-on practice truly helps you to grasp the concepts, it truly helps you automate the knowledge which you've acquired.
-         - Take breaks while learning or studying to alternate between focused and diffuse mode.
+         - Take breaks while learning or studying to alternate between focused and diffused mode.
          - Do not procrastinate the things, use pomodoro method- work for just 20 minutes on something you don’t want to do, then take a break
-         - Test yourself during the studying process by look away and try to recall what you just studied.
+         - Test yourself during the studying process by looking away and try to recall what you just studied.
```
Comment (manaswinidas) [inline learning-to-learn/learning-to-learn.md]: 'trying' to recall

**Comment by tyagipr** [comment]
Yeah sure! @alodahl

**Comment by tyagipr** [comment]
Yes, those slides have a lot of mistakes @alodahl

**Comment by tyagipr** [comment]
Done @alodahl 
Should i do more, as i unassigned from the issue?

**Comment by alodahl** [comment]
@tyagipr please make the changes I requested, then you are welcome to work on another issue. I didn’t mean to unassign you, sorry.

**Comment by tyagipr** [comment]
already done! @alodahl

**Pull request merged**